openwisp_monitoring/monitoring/tests/test_models.py

```diff
@@ -68,6 +70,22 @@ def test_general_write(self):
         measurement = list(query('select * from sync_test').get_points())[0]
         self.assertEqual(measurement['value'], 1)
 
+    def test_device_deleted(self):
```

Comment:
this test can be moved to `openwisp_monitoring.device`, since it's that app that creates the device metrics.

Those tests have helper methods like `_create_device` which you can use.
To create a metric, use `_create_object_metric`.